crates/uv/tests/lock_scenarios.rs

```diff
@@ -1919,7 +1919,7 @@ fn fork_marker_selection() -> Result<()> {
         version = "0.1.0"
         source = { editable = "." }
         dependencies = [
-            { name = "package-a" },
+            { name = "package-a", marker = "sys_platform == 'darwin' or sys_platform == 'linux'" },
```

Comment:
This test (and the one below) is what inspired me to find #5161 and #5162.

Reply:
Unfortunately, the status quo is _correct_, because this update leaves a "hole" (e.g., on Windows) where previously `a` would be installed but now it isn't.